### ISSUE TITLE
Tests fail to run (could be mis-config on my end)

### ISSUE THREAD

**Issue by spicysauce** [dgarlitt/karma-nyan-reporter#6]
I installed karma-nyan-reporter and I'm using Karma v0.12.31 with Jasmine and I do
```
karma start --reporters nyan
```
and all I have in my karma.conf.js file relating to your formatter is:
```
reporters: ['nyan'],
```
and it starts fine, but then when I open another terminal and do:
```
karma run
```
on the "run" window I get:
```
[2015-01-15 08:31:28.546] [DEBUG] config - Loading config /Users/Thien/GitHub/test/www/karma.conf.js

/Users/Thien/GitHub/test/node_modules/karma/lib/runner.js:59
      throw e;
            ^
Error: socket hang up
    at createHangUpError (http.js:1477:15)
    at Socket.socketOnEnd [as onend] (http.js:1573:23)
    at Socket.g (events.js:180:16)
    at Socket.emit (events.js:117:20)
    at _stream_readable.js:944:16
    at process._tickCallback (node.js:442:13)
```
and on the "start" window I get:
```
ERROR [karma]: [TypeError: Cannot call method 'push' of undefined]
TypeError: Cannot call method 'push' of undefined
    at /Users/Thien/GitHub/test/node_modules/karma/lib/reporters/multi.js:8:25
    at Array.forEach (native)
    at addAdapter (/Users/Thien/GitHub/test/node_modules/karma/lib/reporters/multi.js:7:15)
    at null.<anonymous> (/Users/Thien/GitHub/test/node_modules/karma/lib/middleware/runner.js:40:18)
    at g (events.js:180:16)
    at emit (events.js:117:20)
    at schedule (/Users/Thien/GitHub/test/node_modules/karma/lib/executor.js:25:15)
    at /Users/Thien/GitHub/test/node_modules/karma/lib/middleware/runner.js:81:20
    at /Users/Thien/GitHub/test/node_modules/karma/node_modules/connect/node_modules/body-parser/lib/read.js:100:5
    at IncomingMessage.onEnd (/Users/Thien/GitHub/test/node_modules/karma/node_modules/connect/node_modules/body-parser/node_modules/raw-body/index.js:136:7)
```
Is there something in my karma.conf.js or a dependecy that I am missing? I can run my rudimentary tests without the reporter and it works, and I've tried another reporter and it works as well. I'm new to Karma/Jasmine, so I assume it's something on my end that I am doing wrong/missing. Any help would be appreciated, thanks.

**Comment by spicysauce** [comment]
This is my karma.conf.js file:
```
module.exports = function(config) {
  config.set({
    basePath: '',
    frameworks: ['jasmine'],
    files: [
      'lib/ionic/js/ionic.bundle.js',
      'lib/angular-mocks/angular-mocks.js',
      'lib/ngCordova/dist/ng-cordova.js',
      'lib/ionic/js/angular-ui/angular-ui-router.js',
      'lib/angular-animate/angular-animate.js',
      'lib/angular-sanitize/angular-sanitize.js',
      'js/*.js',
      'js/controllers/*.js',
      'js/services/*.js',
      'test/spec/**/*.js'
    ],
    exclude: [
    ],
    preprocessors: {
    },
    reporters: ['nyan'],
    port: 9876,
    colors: true,
    logLevel: config.LOG_INFO,
    autoWatch: false,
    browsers: ['Chrome'],
    singleRun: false
  });
};
```
and this is my package.json:
```
{
  "name": "test",
  "version": "1.0.0",
  "description": "test: An Ionic project",
  "dependencies": {
    "gulp": "^3.5.6",
    "gulp-sass": "^0.7.1",
    "gulp-concat": "^2.2.0",
    "gulp-minify-css": "^0.3.0",
    "gulp-rename": "^1.2.0"
  },
  "devDependencies": {
    "bower": "^1.3.3",
    "gulp-util": "^2.2.14",
    "jasmine-core": "^2.1.3",
    "karma": "^0.12.31",
    "karma-chrome-launcher": "^0.1.7",
    "karma-jasmine": "^0.3.4",
    "karma-nyan-reporter": "0.0.50",
    "shelljs": "^0.3.0"
  }
}
```

**Comment by Nickdouille** [comment]
same here, did you find the solution ?

**Comment by dgarlitt** [comment]
This issue has been resolved in v0.0.60. Thanks for reporting it, @spicysauce sorry for the delay in tending to it.
